packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol

```diff
@@ -68,13 +70,13 @@ contract DisputeGameFactory is OwnableUpgradeable, ISemver {
     GameId[] internal _disputeGameList;
 
     /// @notice Constructs a new DisputeGameFactory contract.
-    constructor() OwnableUpgradeable() {
```

Comment:
what does removing the `OwnableUpgradeable()` constructor do?

Reply:
Honestly, maybe we just use this as a chance to toss the ownable here? The game type modification functions are `onlyProxyAdminOwner` anyway. Better auth structure.

Reply:
Good point, I'm onboard for that, we just have to add a spacer for the old owner slot

Reply:
Changed my mind here, too much diff, let's do this at another time